x-pack/plugins/security_solution/public/common/translations.ts

```diff
@@ -7,6 +7,10 @@
 
 import { i18n } from '@kbn/i18n';
 
+export const SOLUTION_NAME = i18n.translate('xpack.securitySolution.pages.common.solutionName', {
+  defaultMessage: 'Security',
```

Comment:
nit: We have re-defined this same key several times:

Up to you if you want to consolidate the name into 1 area with this PR or add the same name again? I might be missing something but it looks like we have duplicated this same name for translations several times.


<img width="1684" alt="Screen Shot 2021-09-27 at 1 42 26 PM" src="https://user-images.githubusercontent.com/1151048/134974389-72a489e5-029d-4269-94e0-4480043fa6cd.png">

Reply:
good suggestion here, I consolidated some of the translations where I thought it made sense semantically, for instance any translation that was using "Security" as an app or solution name.  In some other areas I left it alone because it seemed more in context with a specific feature.

https://github.com/elastic/kibana/pull/112142/commits/71ef616582bb64bfcc2ae83de5a212e54528eb7f